Frontend/src/pages/crowdactions/Detail/SocialMediaSharing.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { EmailIcon, EmailShareButton, FacebookIcon, FacebookShareButton, FacebookShareCount, LinkedinIcon, LinkedinShareButton, TumblrIcon, TumblrShareButton, TumblrShareCount, TwitterIcon, TwitterShareButton, WhatsappIcon, WhatsappShareButton } from 'react-share';
+import { ICrowdaction } from '../../../api/types';
+
+import styles from './SocialMediaSharing.module.scss';
+
+const SocialMediaSharing = ({ crowdaction }: any) => {
```

Comment:
Instead of casting, you could add:

```
interface ISocialMediaSharingProps {
    crowdaction: ICrowdaction;
}
```

And then:

`const SocialMediaSharing = ({ crowdaction }: ISocialMediaSharingProps) => {`

For an example, see: https://github.com/CollActionteam/CollAction/blob/a454ebfaded866cf8e77cfc3728752a103638c0c/Frontend/src/components/Profile/CrowdactionParticipating.tsx

Then we're actually checking types instead of casting, less chance of runtime issues.